.cargo/config.toml

```diff
@@ -1,2 +1,2 @@
-[build]
-target = "wasm32-wasip1"
+[alias]
+xtask = "run --package xtask --"
```

Comment:
I'm also trying to make `zint-wasm-sys` build for native. This would allow VSCode to work without WASI SDK being installed in `/opt/wasi-sdk`.

It ~works, I just have to figure out how to `cfg(target = "wasm32-wasip1")`, because that doesn't work.